### DISCUSSION TITLE
is there is any possible way to make this work on mac ?

### DISCUSSION THREAD

**Discussion by TheLime1** [thatskymod/Sky-CotL-Scripts#154] [Q&A] [answered]
is there is any possible way to make this work on mac ?

**Comment by thatskymod** [comment]
Not natively. I think your best bet would be to use some kind of emulator (like [MuMu Player](https://www.mumuplayer.com/) for Mac) with Canvas inside.

Or you could try a Windows VM with GPU pass-through—though at that point, getting a Windows PC might make more sense.

I believe this should work, but I don’t own a Mac to test—somebody correct me if I’m wrong.

**Comment by d5s6** [comment, marked as answer]
I just tested it out. It works, but you cannot chat with it. Hard to move around since it doesn't work like how pc or regular mac does. 
**Edit: so chat DOES work,** you just cant see what you're typing :D would recommend a bluetooth/wireless mouse to move around if you're gonna play on the mac forrealz
